src/libraries/System.IO.Compression/ref/System.IO.Compression.cs

```diff
@@ -127,6 +127,7 @@ internal ZipArchiveEntry() { }
         public string Name { get { throw null; } }
         public void Delete() { }
         public System.IO.Stream Open() { throw null; }
+        public System.IO.Stream Open(FileAccess access) { throw null; }
         public System.Threading.Tasks.Task<System.IO.Stream> OpenAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I think we have overlooked something again during API review, we should also have an Async version of the new overload.

I think letting API review board over an email should be enough and we don't need to wait for the response.